index.html

```diff
@@ -38,24 +38,232 @@ <h1 class="hero_title">I'm Reza.<br>Glad to see you!</h1>
+                    don’t hesitate to contact me. 
+                </p>
+                <div class="letsconnect">
+                    <div class="contact-text">Let's Connect</div>
+                        <ul class="social-icons">
+                            <li class="icon">
+                                <a href="https://twitter.com/share_insider"><img src="./icons/twitter.svg" alt="twitter" /></a>
+                            </li>
+                            <li class="icon">
+                                <a href="https://www.linkedin.com/in/mohammad-ali-reza-389717102/"><img src="./icons/Linkedin icon.svg"  alt="linkedin"/></a>
+                            </li>
+                            <li class="icon">
+                                <a href="https://medium.com/@ali.reza.buet" ><img src="./icons/medium.svg" alt="medium"/></a>
+                            </li>
+                            <li class="icon">
+                                <a href="https://github.com/AliRezaBmeDu"><img src="./icons/github.svg"  alt="github"/></a>
+                            </li>
+                            <li class="icon">
+                                <a href="https://wellfound.com/u/mohammad-ali-reza" ><img src="./icons/angellist.svg" alt="angellist"/></a>
+                            </li>
+                        </ul>
+                        <button class="resume-button">Get my resume</button>
+                    </div>
+                    
+                </div>
+
+            <div class="languages">
+                <div class="languagebar">
+                    <h2>Languages</h2>
+                    <button class="lang-button">&gt;</button>
+                </div>
+                
+                <ul class="language-skill">
+                    <li>
+                        <div class="skill">
+                            <img src="./images/javs.png"/> 
+                            <p class="name-lang">Javascript</p>
+                        </div>
+                    </li>
+                    <li>
+                        <div class="skill">
+                            <img src="./images/htm.png"/> 
+                            <p class="name-lang">HTML</p>
+                        </div>
+                    </li>
+                    <li>
+                        <div class="skill">
+                            <img src="./images/design.png"/> 
+                            <p class="name-lang">CSS</p>
+                        </div>
+                    </li>
+                </ul>
+            </div>
+            
+            <div class="frameworks">
+                <h2>Frameworks</h2>
+                <button class="right">&gt;</button>
+            </div>
+            <div class="hr"></div>
+            <div class="frameworks">
+                <h2>Skills</h2>
+                <button class="right">&gt;</button>
```

Comment:
- Please can you make use of an icon or the image of Chevron in the Figma design, using the `&gt` does not add aesthetics to your portfolio, please use a proper Chevron icon for this.  

You can decide to use font awesome icons, here is a setup [link](https://fontawesome.com/docs/web/add-icons/how-to)

##### Figma design 
![image](https://github.com/AliRezaBmeDu/Portfolio-Website/assets/58771507/b7f39a4f-63ab-486c-8dbc-502e09b5cf91)


##### Your design
![image](https://github.com/AliRezaBmeDu/Portfolio-Website/assets/58771507/6977a489-930c-440c-be3b-b1fc89a3acd3)